AudibleBoulders/AudibleBoulders
# Add spyupdate to the init script and update zipfile

#### What's this PR do?
Add spyupdate to the init script and update zipfile

#### What are the important parts of the code?
gitSpyInit

#### How should this be tested by the reviewer?
Add a new dashboard run go through the setup instructions!
You should automatically be setup after you run the init script

#### Is any other information necessary to understand this?
N/A

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
N/A

#### Screenshots (if appropriate)
N/A
